test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeModule.cs

```diff
@@ -34,7 +34,7 @@ public EdgeModule(string id, string deviceId, IotHub iotHub)
 
         public static Task WaitForStatusAsync(IEnumerable<EdgeModule> modules, EdgeModuleStatus desired, CancellationToken token)
         {
-            string[] moduleIds = modules.Select(m => m.Id).Distinct().ToArray();
+            string[] moduleIds = modules.Select(m => m.Id.TrimStart('$')).Distinct().ToArray();
```

Comment:
Isn't it better to store it with the identifier?

Reply:
When we're talking about a system module (starts with `$`), most of the time we need to preserve that leading dollar sign. There's only one case where we need to drop it; that's when we're matching up module names with docker container names. But in `EdgeModule` previously we were storing the module name _without_ the dollar sign, then adding it back when we needed it (which was most of the time). I just reversed the behavior, but the net effect is the same.